user-context.md

```diff
@@ -20,6 +20,10 @@ web:
     uri: "/me"
 ```
 
+### Authentication
+
+This endpoint requires authentication, and accepts all forms of authentication. 
```

Comment:
I'm fine with this only accepting OAuth tokens as well, but wrote it in this way to get started. `/me` isn't that useful to be honest.